WeakAuras/Types.lua

```diff
@@ -1490,7 +1490,7 @@ if WeakAuras.IsRetail() then
   Private.GetCurrencyIDFromLink = C_CurrencyInfo.GetCurrencyIDFromLink
   Private.ExpandCurrencyList = C_CurrencyInfo.ExpandCurrencyList
   Private.GetCurrencyListInfo = C_CurrencyInfo.GetCurrencyListInfo
-elseif WeakAuras.IsWrathClassic() then
+elseif WeakAuras.IsWrathOrCata() then
```

Comment:
WeakAuras.race_types needs to be checked, cata added worgen and goblin

Reply:
They are on load options

Reply:
it's good 
![image](https://github.com/WeakAuras/WeakAuras2/assets/189656/0ed8d3b9-c356-4a5c-be18-c1a799bacd6f)